src/resources/Charges.ts

```diff
@@ -42,6 +42,7 @@ export enum ChargeThreeDsMode {
     FORCE = "force",
     SKIP = "skip",
     IF_AVAILABLE = "if_available",
+    REQUIRE = "require",
```

Comment:
Technically speaking, there is 1 more called `provided`, but it is a read only value, meaning to say, you can never set that value in the request. You can only receive as a response. You might want to add it if parsing a non existing enum value will cause your code to crash.

Reply:
sure! sounds good! Better have it there to be on the safe side